cmd/export.go

```diff
@@ -155,22 +169,23 @@ func (o *ExportOptions) Run() error {
 	}
 	path = filepath.Join(path, dsr.Name)
 
-	// TODO: Implement flags specified in the RFC 0014, only set the zip flag if the export
-	// includes multiple files and the --directories-for-files flag is not true.
-	o.Zipped = true
-	if o.Zipped {
-		dst, err := os.Create(fmt.Sprintf("%s.zip", path))
-		if err != nil {
-			return err
-		}
+	// TODO (dlong): When --zipped flag is supported, don't always do this.
+	dst, err := os.Create(fmt.Sprintf("%s.zip", path))
+	if err != nil {
+		return err
+	}
 
-		if err = dsutil.WriteZipArchive(o.Repo.Store(), ds, res.String(), dst); err != nil {
-			return err
-		}
-		return dst.Close()
+	// TODO (dlong): Use --body-format here to convert the body and ds.Structure.Format, before
```

Comment:
I'm SO pumped for this to land, would love it if we landed a small placeholder for this then started to think about piped / streaming use-cases for export with conversions.

The issue of converting to CSV has been on my mind for a whioe. We'll have to take care about down-depth conversions (say, JSON of depth 5 to csv). I think we should error on conversions to CSV when dataset depth is either not present or above 2. On the upside we can rely on depth now to be smarter